express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+let grades = {};
+const arrayResponse = [];
+
+app.get('/api/grades', (req, res) => {
+  res.json(arrayResponse);
+});
```

Comment:
For now let's keep the data model as an object and then generate the array on-demand in the GET route callback.